stdlib/websub/src/main/ballerina/websub/Module.md

```diff
@@ -245,3 +245,104 @@ where the values specified via the Config API would override values specified as
 | b7a.websub.hub.signaturemethod | "SHA256"      | The signature method to use for authenticated content distribution |
 | b7a.websub.hub.remotepublish   | false         | Whether publishing updates against the topics in the hub could be done by remote publishers via HTTP requests with `hub.mode` set to `publish`  |
 | b7a.websub.hub.topicregistration | true      | Whether a topic needs to be registered at the hub for publishers to publish updates against the topic and for subscribers to send subscription requests for the topic |
+
+## Introducing Specific Subscriber Services (Webhook Callback Services)
+
+Ballerina's WebSub subscriber service listener can be extended to introduce specific Webhooks.
+ 
+With specific subscriber services, multiple resources can be introduced to accept content delivery requests, instead
+ of the single `onNotification` resource. These resources would correspond to the content delivery requests that will 
+ be delivered for a particular topic.
+ 
+For example, assume by subscribing to a particular topic with an issue tracking system, you receive notifications 
+either when an issue is opened or when an issue is closed. With a custom subscriber service listener which extends the 
```

Comment:
With a custom subscriber service listener, which extends the 
generic WebSub subscriber service listener, you can allow two resources to accept content delivery requests (e.g., `onIssueOpened` 
and `onIssueClosed`) instead of the `onNotification` resource.